admin/app/components/organizations/target-profiles-section.hbs

```diff
@@ -45,11 +45,13 @@
                   </LinkTo>
                 </td>
                 {{#if this.accessControl.hasAccessToOrganizationActionsScope}}
-                  <td>
-                    <PixButton @backgroundColor="red" @size="small" @triggerAction={{(fn this.openModal summary)}}>
-                      Détacher
-                    </PixButton>
-                  </td>
+                  {{#unless summary.isPublic}}
```

Comment:
`#if (and this.accessControl.xxxx (not summary.isPublic))` ou un **joli** getter

Reply:
https://github.com/1024pix/pix/pull/7528/files#diff-8f7d1a9c7bbd439aa9f96fd22f21783f6bad587247802bd0fd576b255650f45cR22